AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -677,3 +682,27 @@ extension MeetingModel: EventAnalyticsObserver {
         return jsonDict
     }
 }
+
+// MARK: TranscriptEventObserver
+
+extension MeetingModel: TranscriptEventObserver {
```

Comment:
UI will be done by others?

Reply:
That was the plan before, but the latest plan is I will implement the demo UI. Will update this PR with demo app changes once ready.

Reply:
Added UI implementation